locale/pl/LC_MESSAGES/client.po

```diff
@@ -155,15 +155,15 @@ msgstr "Otwórz Gmail"
 
 #: app/scripts/views/mixins/open-webmail-mixin.js:25
 msgid "Open Hotmail"
-msgstr ""
+msgstr "Otwórz Hotmail"
 
 #: app/scripts/views/mixins/open-webmail-mixin.js:31
 msgid "Open Yahoo"
-msgstr ""
+msgstr "Otwórz Yahoo"
 
 #: app/scripts/views/mixins/open-webmail-mixin.js:37
 msgid "Open Outlook"
-msgstr ""
+msgstr "Otwórz Outlook"
 
```

Comment:
_Otwórz Gmaila, Hotmaila, Yahoo, Outlooka_ or _Otwórz serwis_ with domain (not name)